cypress/e2e/shared/deepLinks.test.ts

```diff
@@ -0,0 +1,120 @@
+import {Organization} from '../../../src/types'
+
+describe('Deep linking', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+  })
+
+  // If you're here and the test failure you're looking at is legitimate, it probably means a page
+  // that a deep link pointed to has changed. There are implications to this; docs and marketing pages
+  // might use the deep link that has changed. We want to avoid having broken deep links out on the web,
+  // so there might be some follow-up work necessary with docs or marketing pages.
+  it.skip('should be redirected to the approprate page from a shortened link', () => {
```

Comment:
I'm about to go on PTO for 6 work days. I'm skipping these so that any flakiness in them doesn't disrupt the UI team.